dev/fattest.simplicity/src/componenttest/rules/repeater/RepeatTests.java

```diff
@@ -172,6 +186,20 @@ public void evaluate() throws Throwable {
             MultipleFailureException.assertEmpty(errors);
         }
 
+        private static boolean runAdditionalEvalCheck(Optional<Supplier<Boolean>> additionalEvaluationCheck) {
+
+            final String m = "runAdditionalEvalCheck";
+
+            if (additionalEvaluationCheck.isPresent()) {
+                Log.info(c, m, "Running additional eval check");
+                boolean result = additionalEvaluationCheck.get().get();
+                Log.info(c, m, "Additional eval check result: " + result);
```

Comment:
I don't think we need info logging for all checks. Maybe it would make sense to log if a check fails, though we'll already get the `"Skipping <test> with action: <action>"` message.